README.md

```diff
@@ -71,50 +71,46 @@ Once you've opened the project in [Codespaces](#github-codespaces), [Dev Contain
 
 3. Configure your environment variables that will be used for deployment:
 
-> [!IMPORTANT]
-> Please note that SQL Auth cannot be turned off as part of the deployment. However, an admin name and password must be given as part of the database creation process. In the current state, you can enable Microsoft Entra authentication only via the portal.
-
- * aad_admin_name: Principal name of the external administrator (UPN). If you need help finding this value, please login with the Azure CLI and run the script: [fetch-principal-info.sh](./scripts/fetch-principal-info.sh) or [fetch-principal-info.ps1](scripts/fetch-principal-info.ps1)
+    > [!IMPORTANT]
+    > This project code uses passwordless authentication with the Azure SQL server, but it doesn't currently turn off SQL password auth entirely, due to an issue with Bicep-based deployments. The username is set to a unique string, and the password is set to an auto-generated value. Once deployed, you can disable SQL password auth via the Azure portal.
 
-    ```shell
-    azd env set AZURE_PRINCIPAL_NAME yourprincipalname
-    ```
+    For the passwordless authentication to be properly set up, you must set the principal name of the external administrator (UPN). If you need > help finding this value, please login with the Azure CLI and run the script: [./scripts/fetch-principal-info.sh](./scripts/ fetch-principal-info.sh) or [./scripts/fetch-principal-info.ps1](scripts/fetch-principal-info.ps1)
 
- * administrator_login: login name of the administrator for the Azure SQL Database.
+    Once you know your principal name, set it as an azd environment variable:
 
     ```shell
-    azd env set AZURE_ADMIN_LOGIN someadminname
+    azd env set AZURE_PRINCIPAL_NAME yourprincipalname
     ```
 
- * administratorPassword: password of the administrator for the Azure SQL Database.
+4. Deploy the resources:
```

Comment:
It's missing azd up right now, I think that got lost in a merge at some point maybe